src/Profiles.ts

```diff
@@ -253,6 +272,22 @@ export class Profiles {
     public async refresh(): Promise<void> {
         this.allProfiles = [];
         this.allTypes = [];
+
+        // TODO: Add Base ProfileType in registeredApiTypes
+        // This process retrieves the base profile if there's any and stores it in an array
+        // If base is added in registeredApiType maybe this process can be removed
+        const profileManagerA = await this.getCliProfileManager("base");
+        if (profileManagerA) {
+            try {
+                const baseProfile = await profileManagerA.load({ loadDefault: true });
+                this.allProfiles.push(baseProfile);
+            } catch (err) {
+                if (!err.message.includes(`No default profile set for type "base"`)) {
+                    vscode.window.showInformationMessage(err.message);
+                }
+            }
+        }
+
```

Comment:
@phaumer @JillieBeanSim @lauren-li how can I add base in `registerApiType()`? I think line 275-290 can be removed if base is registered as an api type.